### PR TITLE
fix blank bug, add tests

### PR DESCRIPTION
**Pull request by davidkircos**

### CODE REVIEW

File: quadratic-kernels/python-wasm/quadratic_py/utils.py [hunk 1]
```diff
@@ -135,7 +142,7 @@ def to_python_type(value: str, value_type: str) -> int | float | str | bool:
         elif value_type == "logical":
             return ast.literal_eval(normalize_bool(value))
         elif value_type == "instant":
-            return datetime.utcfromtimestamp(int(value))
+            return datetime.fromtimestamp(int(value), tz=pytz.utc)
```
Comment (davidkircos) [inline quadratic-kernels/python-wasm/quadratic_py/utils.py:145]: This fixes a complaint when you run python tests

File: quadratic-kernels/python-wasm/tests/test.py [hunk 2]
```diff
@@ -236,8 +238,35 @@ def test_to_python_type(self):
         assert to_python_type("abc123", "text") == "abc123"
 
         # instant
-        assert to_python_type("1352505600", "instant") == pd.Timestamp("2012-11-10 00:00:00")
-        assert to_python_type("1352518200", "instant") == pd.Timestamp("2012-11-10 03:30:00")
+        assert to_python_type("1352505600", "instant") == pd.Timestamp("2012-11-10 00:00:00+00:00")
+        assert to_python_type("1352518200", "instant") == pd.Timestamp("2012-11-10 03:30:00+00:00")
```
Comment (davidkircos) [inline quadratic-kernels/python-wasm/tests/test.py:242]: add tz info

**Review by ddimaria — APPROVED** [review]
Code looks good!

File: quadratic-kernels/python-wasm/quadratic_py/utils.py [hunk 3]
```diff
@@ -26,7 +28,12 @@ def __bool__(self):
         return False
     
     def generic_overload(self, other, op):
-        return op(0, other)
+        if isinstance(other, type(None)):
+            return op(None, other)
+        elif isinstance(other, str):
+            return op("", other)
+        else:
+            return op(0, other)
```
Comment (davidkircos) [inline quadratic-kernels/python-wasm/quadratic_py/utils.py:36]: This stack order feels weird. 

Just assuming is a number in the else case. Not sure how we could change it though

**Pull request merged**